app/Models/Permission.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace App\Models;
+
+use App\Models\Concerns\DefinesPermissions;
+use Spatie\Permission\Models\Permission as SpatiePermission;
+
+class Permission extends SpatiePermission
+{
+    use DefinesPermissions;
```

Comment:
Is this perhaps one that should be overwritten? Since we would really only ever want the `index` and `show` permission for this?